src/Common/src/System/Globalization/IdnMapping.cs

```diff
@@ -36,6 +36,8 @@ namespace System.Globalization
 #endif
     sealed partial class IdnMapping
     {
+        private const int StackallocThreshold = 512;
+
```

Comment:
whoops - would you prefer I modified the windows implementation or move this to the unix implementation